src/isDate.spec.js

```diff
@@ -0,0 +1,9 @@
+import isDate from './isDate';
+
+// TODO how to test the platform differences?
+
+describe('isDate', function() {
+  it('accepts Date object', () => {
+    expect(isDate(new Date())).toBe(true);
+  });
+});
```

Comment:
This is probably unfinished...

Reply:
Yes, I do not know how to correctly test this function as its implementation is platform dependent.

Reply:
Please recheck, I completed the testcase @dmitrymarokhonov